Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -42,9 +47,17 @@ protected static IMixedRealityInputSystem InputSystem
             }
         }
 
-        // list of pointers
-        protected List<IMixedRealityPointer> pointers = new List<IMixedRealityPointer>();
-        public List<IMixedRealityPointer> Focusers => pointers;
+        protected List<IMixedRealityPointer> focusingPointers = new List<IMixedRealityPointer>();
```

Comment:
I would make this readonly and clear it as needed instead of recreating it. Same for pressingInputSources.

Reply:
Out of curiosity, why?

Reply:
Fixed.

Reply:
To avoid a new allocation each time.